sunpy/image/slit.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
+
+import astropy.units as u
+
+
+def slit(mcube_in, range_a, range_b, N_slits=0, shift_x=+1, shift_y=-1,):
+    """
+    Returns an array with intensity along the slit on the y axis and time
```

Comment:
Now that this handles slits of different widths, I would expand this to give a more full description of what the function does.